lib/manager/buildkite/extract.spec.ts

```diff
@@ -31,5 +31,10 @@ describe('manager/buildkite/extract', () => {
       expect(res).toMatchSnapshot();
       expect(res).toHaveLength(2);
     });
+    it('extracts git-based plugin with .git at the end of its name', () => {
+      const res = extractPackageFile(Fixtures.get('pipeline6.yml')).deps;
+      expect(res).toMatchSnapshot();
```

Comment:
Should I set up an another fixture with the expected result? All the other tests here just use snapshots.

Reply:
We prefer now to use explicit matcher instead of snapshot tests. If the result object is huge, we are accepting snapshots, but those tests also heed to have some explicit matchers, so failures can't pass via silent snapshot updates.

Jest snapshot function also accepts partial object matching. Check jest docs or view other renovate tests about. 😉